docs/testing-with-codeception.md

```diff
@@ -744,3 +745,20 @@ Projects can use a custom Codeception configuration file and override Altis' zer
 ```sh
 composer dev-tools codecept run -- -c path/to/codeception.yml
 ```
+
+## FAQ
+
+- Why do my tests fail because it cannot find the content I created using WPBrowser DB helper functions ?
+
+WPBrowser DB helper functions, like `$I->havePostInDatabase()` and the alike, uses direct database queries to manage content, which means WordPress filters are not run for those operations, so integrations like ElasticPress are not notified of the changes and do not update the Elastic index as a result of that. So while the content is created in the database, it is not synced to Elastic index, and subsequently will not show up in queries that are typically run by Elastic.
```

Comment:
```suggestion
WPBrowser DB helper functions, like `$I->havePostInDatabase()` and the like, use direct database queries to manage content, which means WordPress filters are not run for those operations. This means that integrations like ElasticPress are not notified of the changes and do not update the Elasticsearch index as a result of that. So while the content is created in the database, it is not synced to Elasticsearch, and subsequently will not show up in queries that are handled by ElasticPress.
```

Nitpick but "Elastic" can be a bit broad in infrastructure terms as there are things like Elasticache (Redis) that may get discussed too. Better to be explicit I think.

Reply:
I had been under the impression the product changed name to Elastic rathe than Elasticsearch, going through docs now that seems to be wrong 🤦 . Rest of changes are legit 👍 got it.

Reply:
Ohh sorry yeah - the company behind it did though the software is still Elasticsearch, and to confuse matters further AWS's fork is called OpenSearch!

Reply:
Yeah, that's what confused me then!